src/components/PrimaryFeatures.jsx

```diff
@@ -129,7 +129,7 @@ export function PrimaryFeatures() {
                       <Image
                         className="w-full"
                         src={feature.image}
-                        alt=""
+                        alt={feature.title}
```

Comment:
wouldn't this be repetitive as the screen reader would first read this and then read the title which is a header anyways

Reply:
I will check it out and make corrections